react/features/deeplinking/actions.js

```diff
@@ -0,0 +1,30 @@
+/* @flow */
+
+import { appNavigate } from '../app';
+
+import { OPEN_DESKTOP_APP, OPEN_WEB_APP } from './actionTypes';
+
+/**
+ * Continue to the conference page.
+ *
+ * @returns {Function}
+ */
+export function openWebApp() {
+    return (dispatch: Dispatch<*>) => {
+        dispatch({ type: OPEN_WEB_APP });
+        dispatch(appNavigate());
```

Comment:
What is the additional appNavigate for? The feature works by setting a boolean in redux to navigate to web and then calling appNavigate? It's not easy to bypass this two step approach?

Reply:
The additional appNavigate is to enter into the conference! Why do you want to bypass the 2-step approach?

Reply:
I don't understand the need for a two step approach. It can't be done in one step? It seems like a one step process. I expect, only for example, something like `openWebApp` to handle everything. That's why I'm asking if there was no easy way to bypass a two step approach.